packages/cli-lib/lang/en.lyaml

```diff
@@ -2,706 +2,726 @@ en:
   cli:
     commands:
       accounts:
-        describe: "Commands for working with accounts."
+        describe: 'Commands for working with accounts.'
```

Comment:
It seems like prettier replaced all double quotation marks with singles--was this intentional?